package.json

```diff
@@ -22,7 +22,9 @@
     "debug": "^2.1.1",
     "loopback-connector": "^2.1.0",
     "loopback-oracle-installer": "~2.0.0",
-    "strong-globalize": "^2.6.2"
+    "strong-globalize": "^2.6.2",
+    "node-oracledb": "^1.11.0"
```

Comment:
There are two issues here:
- The module name is `oracledb`. See https://www.npmjs.com/package/oracledb
- At the moment, we pull in the `oracledb` dependency via `loopback-oracle-installer` which downloads oracle instant client and pre-built oracledb based on the client platform and node version.